config/environments/production.rb

```diff
@@ -54,17 +54,23 @@
   # Use a different logger for distributed setups.
   # config.logger = ActiveSupport::TaggedLogging.new(SyslogLogger.new)
 
-  module ActiveSupport::TaggedLogging::Formatter
-    def call(severity, time, progname, data)
-      data = { msg: data.to_s } unless data.is_a?(Hash)
-      tags = current_tags
-      data[:tags] = tags if tags.present?
-      _call(severity, time, progname, data)
-    end
+  # config/environments/production.rb
+  config.lograge.enabled = true
+  config.lograge.custom_options = lambda do |event|
+    options = event.payload.slice(:request_id, :user_id)
+    options[:params] = event.payload[:params].except("controller", "action")
+    options
+  end
+  config.lograge.formatter = Lograge::Formatters::Json.new
+  
+  # app/controllers/application_controller.rb
```

Comment:
as per the comment, this method should be added to `app/controllers/application_controller.rb` (there shouldn't be any method definition in `config/environments/*.rb`)... and probably this should be called as an `after_filter` for all requests, otherwise it's just a defined method that will never be called.